network/base/admin.py

```diff
@@ -9,7 +9,7 @@ class AntennaAdmin(admin.ModelAdmin):
 
 
 class StationAdmin(admin.ModelAdmin):
-    list_display = ('name', 'owner', 'lng', 'lat', 'location')
+    list_display = ('name', 'owner', 'lng', 'lat')
```

Comment:
You could expose `qthlocator` on the admin interface by replacing `location` here.